theseus/Cargo.toml

```diff
@@ -31,6 +31,8 @@ thiserror = "1.0"
 tracing = "0.1"
 tracing-error = "0.2"
 
+tauri = { version = "1.2" }
```

Comment:
would make this a feature flag so the playground (and eventually the CLI) works without issue